eq-author-api/middleware/convertQuestionnaire.js

```diff
@@ -0,0 +1,46 @@
+const { getQuestionnaire } = require("../db/datastore");
+const { logger } = require("../utils/logger");
+const request = require("request");
+
+module.exports = async (req, res) => {
+  const usingGCP = process.env.DATABASE === "firestore";
+  const CONVERSION_URL = process.env.CONVERSION_URL;
+  if (!usingGCP) {
+    res
+      .status(503)
+      .json({ error: "Endpoint not supported when using AWS configuration" });
+  } else if (!CONVERSION_URL) {
+    res.status(500).json({
+      error: "Cannot convert questionnaire without CONVERSION_URL env var set",
+    });
+  } else {
```

Comment:
Minor point. Don't need the "else" and the "else if" as the module would exit when you call res.status.json anyway.